packages/docs/docs/docs/migrate_from_1.x_to_2.x.md

```diff
@@ -0,0 +1,117 @@
+
+#### Check
+
+```
+Usage: check [contract] [options]
+
+checks your contracts for potential issues
+
+Options:
+  --skip-compile  skips contract compilation
+  -h, --help      output usage information
+```
+
+As discussed in the ["Safety checks" section of the documentation](https://docs.zeppelinos.org/docs/advanced.html#safety-checks), the zOS CLI performs a series of safety checks on your contracts as a sub-routine of some of it's commands. The new `check` command allows you to perform these checks in a standalone manner.
+
+#### Set-admin
+
+```
+Usage: set-admin [alias-or-address] [new-admin-address] --network <network> [options]
+
+change upgradeability admin of a contract instance. Provide the [alias] or [package]/[alias] of the contract to change the ownership of all its instances, or its [address] to change a single one. Note that if you transfer to an incorrect address, you may irreversibly lose control over upgrading your contract.
+
+Options:
+  -y, --yes                accept transferring admin rights (required)
+  -n, --network <network>  network to be used
+  -f, --from <from>        specify transaction sender address
+  --timeout <timeout>      timeout in seconds for each blockchain transaction (defaults to 600s)
+  -h, --help               output usage information
+```
+
+Convenience command that allows you to change the address that can perform upgrades on a proxy.
+
+#### Unlink
+
+```
+Usage: unlink [dependencyName1 ... dependencyNameN]
+
+unlinks libraries from the project. Provide a list of whitespace-separated library names
+
+Options:
+  --push [network]     push all changes to the specified network
+  --skip-compile       skips contract compilation
+  -f, --from <from>    specify the transaction sender address for --push
+  --timeout <timeout>  timeout in seconds for each blockchain transaction (defaults to 600s)
+  -h, --help           output usage information
+```
+
+Opposite to the `link` command.
+
+## Modified commands
+
+In 2.x, there are minor changes to the commands from 1.x, like some of them now having options like `--skip-compile`, but there are also more signiffcant changes.
+
+*`bump`* is now only relevant for published EVM packages, and is no longer applicable to projects that are not intended for reusability.
+
+*`link`* can now be called for multiple EVM packages.
+
+*`init`* no longer has the `--lib` option.
+
+## Compatibility
+
+Unfortunately, 2.x introduces breaking changes, so a project that was created using 1.x cannot be upgraded to 2.x. This means that if you upgrade your global `zos` NPM package to 2.x, it won't be compatible with projects that were created using the old version of `zos`. In such cases, the CLI will detect the incompatibility and warn you. This is one of the reasons why `npx zos` is recommended over `zos` usage, so that each NPM project can target its own `zos` version.
```

Comment:
> that was created using 1.x cannot be upgraded to 2.x

Cannot be _automatically_ upgraded to 2.x. There is a good chance that we'll set up a manual migration guide/script in the near future.